client/src/services/DonuService.ts

```diff
@@ -72,6 +71,7 @@ export const fetchDonuModels = async (): Promise<Model.Model[]> => {
           edges: model.graph.edges,
         },
       };
+      delete model.gromet;
```

Comment:
Why is this needed?

Reply:
my understanding of how garbage collection works in `js` is that when the function finishes, the `model` object will be deleted by the garbage collector which includes the `gromet` property, so this deletion should not be necessary as its done automatically. I am not 100% certain though, I can test this out tomorrow by allocating like a 3gb buffer in the `model` object and see if the memory is released after the function is finished.

Reply:
i did the buffer allocation test and i compared js heap size before and after adding `delete model.gromet`, doesnt look like this line is necessary

Reply:
Awesome, thank you @mj3cheun